README.md

```diff
@@ -1,3 +1,36 @@
 # Knowledge Base Solution Pack
 
-Documentation work is in progress!!
+## Release Information
+
+- Solution Pack Version: 1.0.1
+- Minimum Compatible FortiSOAR™ Version: 7.2.0
+- Authored By: Fortinet
+- Certified: No
+
+## Overview
+
+*Knowledge Base* solution pack is designed to provide best practices to configure and use FortiSOAR™ in an optimal manner. The FortiSOAR™ Knowledge Base Solution Pack provides the user with information about different things like (triage process, tools etc) used in a SOAR.
```

Comment:
*Knowledge Base* solution pack is designed to provide best practices to configure and use FortiSOAR™ in an optimal manner.  --> I don't think, this is needed.

Reply:
The FortiSOAR™ Knowledge Base Solution Pack provides the user with information about different things like (triage process, tools etc) used in a SOAR. --> This Solution Pack provides knowledge about best practices, triage process, tools, etc. used in a FortiSOAR.